external/js/cothority/tsconfig.spec.json

```diff
@@ -0,0 +1,12 @@
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    // Specify the root directory of input files. Use to control the output directory structure with --outDir.
+    "rootDir": "./",
+    "baseUrl": "./"
```

Comment:
how much adding the `strict` flag makes eveything explose?

Reply:
Didn't try that - but that's for another day ;) My goal here is not to revamp the whole of dedis/cothority, but to add a new layer (and hope the bottom layers will disappear someday ;)